VHbbAnalysis/Heppy/python/VHGeneratorAnalyzer.py

```diff
@@ -154,7 +154,7 @@ def makeMCInfo(self, event):
         event.genallbquarks = []
         event.genwzquarks = []
         event.gentopquarks  = []
-        event.genallstatus2bhadrons = [ p for p in event.genParticles if p.status ==2 and hasBottom(p.pdgId()) ]
+        event.genallstatus2bhadrons = [ p for p in event.genParticles if p.status() ==2 and self.hasBottom(p.pdgId()) ]
```

Comment:
it was a series of errors: hasBottom without self was just wrong but was not complaining because it never got there due to the missing () in p.status .... now is fully tested and fixed